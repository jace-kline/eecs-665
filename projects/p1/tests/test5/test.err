FATAL [1,78]: Illegal character \
