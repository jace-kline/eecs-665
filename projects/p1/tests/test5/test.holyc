"This is a test of escaping inside a string literal with an additional space \ n Hello testing testing"