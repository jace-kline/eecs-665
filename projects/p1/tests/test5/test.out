ID:his [1,3]
ID:is [1,7]
ID:a [1,10]
ID:test [1,12]
ID:of [1,17]
ID:escaping [1,20]
ID:inside [1,29]
ID:a [1,36]
ID:string [1,38]
ID:literal [1,45]
ID:with [1,53]
ID:an [1,58]
ID:additional [1,61]
ID:space [1,72]
ID:n [1,80]
ID:Hello [1,82]
ID:testing [1,88]
ID:testing [1,96]
EOF [1,103]
